examples/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ set(EXAMPLES_EXEC_LIST
 set(EXAMPLES_LIST
     ${EXAMPLES_EXEC_LIST}
     custom-stopping-criterion
+    file-config-solver
```

Comment:
Does this mean they are not run as part of our CI?
Is there a reason why we don't?

Reply:
I have added test which only checks it can run properly.

Reply:
I do not reuse the existing one because it needs to accept additional parameter